import/marksessions.php

```diff
@@ -66,12 +66,12 @@
 
 $formparams = ['id' => $cm->id,
                'sessionid' => $pageparams->sessionid,
-               'grouptype' => $pageparams->grouptype];
+               'grouptype' => $pageparams->grouptype, ];
 $form = null;
 if (optional_param('needsconfirm', 0, PARAM_BOOL)) {
     $form = new \mod_attendance\form\import\marksessions($url->out(false), $formparams);
 } else if (optional_param('confirm', 0, PARAM_BOOL)) {
-    $importer = new \mod_attendance\import\marksessions(null, $att, null, null, $importid);
```

Comment:
looks like here'e the matching change for the function parameter order change - I think we should probably do this as it's own commit rather than bundling into the "codechecker fixes" if there's a good reason we should do this.